### PR TITLE
Ipcheck 0.2 - stable enough to publish.

### PR DESCRIPTION
**Pull request by mihakralj**
Let me know if I missed any of the files...

### CODE REVIEW

File: net-mgmt/ipcheck-community/+POST_INSTALL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+if /usr/local/etc/rc.d/configd status > /dev/null; then
```
Comment (mimugmail) [inline net-mgmt/ipcheck-community/+POST_INSTALL:1]: Why do we need this file?

Comment (mihakralj) [reply net-mgmt/ipcheck-community/+POST_INSTALL:1]: Adding refresh of configd to +POST_INSTALL is required to register all new API endpoints that are registered by ipcheck.

I just followed the best practice at the end of a Helloworld plugin documentation: https://docs.opnsense.org/development/examples/helloworld.html

Comment (mimugmail) [reply net-mgmt/ipcheck-community/+POST_INSTALL:1]: Ok, I'll pull it, build a pkg and test. Thx :)

**Comment by mimugmail** [comment]
Just a small cleanup:
https://github.com/mimugmail/opn-repo/commit/5715ba4197174ca9971bb7c8ec62d1fc59e13ec0

Looks good, I'll add the plugin to the repo tomorrow ( my timezone ;) )
Thx!

**Pull request merged**